javascript/javascript1/week2/exercise-1.js

```diff
@@ -0,0 +1,31 @@
+// Exercise 1: Flight booking fullname function
+
+function getFullName(firstname, surname, useFormalName = false, gender) {
+	if (!firstname.trim() && !surname.trim()) {
```

Comment:
very nice check! you can also use `typeof` to check if the arguments are for sure of type string. Uless of course it's 007 that flies this plane 😉